plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -0,0 +1,8 @@
+return {
+    -- list your feeds here:
+    -- only supports http URL for now
+    -- Atom is currently not supported, only RSS
+    -- set limit to "0" means no download, "-1" no limit.
+    { "http://www.huffingtonpost.com/feeds/index.xml", limit = 2 },
```

Comment:
As we not rely on "description" but rather url link to html document it shouldn't be so hard.

Just need to find RSS format (as atom is yet not supported/fully tested) __neutral__ news source if that would be too hard let's take a sport source.